### ISSUE TITLE
Use Coturn Without Public IP

### ISSUE THREAD

**Issue by minghaow** [coturn/coturn#642]
I have to run coturn in a Local Area Network without internet access. Therefore I do not have a public ip for coturn server. Can it work in this kind of network environment? What do I need to do?

Thanks a lot!

**Comment by chanduthedev** [comment]
Hi @minghaow, Can you share the purpose of the Coturn you are using for? Generally Coturn will be used if both the end points not reachable directly or there is direct connection between two peers(involving firewalls in the network). But in your case, you menstioned that you are going to use in local area network where all the nodes are reachable and Coturn will be of no use even if you configured.

**Comment by misi** [comment]
I agree with @chanduthedev. I don't see too much reason for it, but in theory it is possible to use it..
